doc/input-output-reference/src/overview/group-thermal-zone-description-geometry.tex

```diff
@@ -3114,6 +3117,14 @@ \subsubsection{Surface Inside Face Lights Radiation Heat Gain Energy {[}J{]}}\la
 
 These ``inside face lights radiation heat gain'' output variables describe the heat transferred by shortwave radiation onto the inside face. The values are always positive and indicate heat is being added to the surface's face by shortwave radiation that emanated from electric lighting equipment and was absorbed by the surface. Different versions of the report are available including the basic heat gain rate (W), and a per unit area flux (W/m2), and an energy version (J).
 
+\subsubsection{Surface Inside Face Lights Radiation Heat Gain Rate From Other Zones {[}W{]}}\label{surface-inside-face-lights-radiation-heat-gain-rate-from-other-zones-w}
+
+\subsubsection{Surface Inside Face Lights Radiation Heat Gain Rate per Area From Other Zones {[}W/m2{]}}\label{surface-inside-face-lights-radiation-heat-gain-rate-per-area-from-other-zones-wm2}
+
+\subsubsection{Surface Inside Face Lights Radiation Heat Gain Energy From Other Zones {[}J{]}}\label{surface-inside-face-lights-radiation-heat-gain-energy-from-other-zones-j}
+
+These ``inside face lights radiation heat gain (from other zones)'' output variables describe the heat transferred by shortwave radiation onto the inside face as a result of lights from other zones being transmitted through interzone windows. The values are always positive and indicate heat is being added to the surface's face by shortwave radiation that emanated from electric lighting equipment in other zones that goes through an interzone window and was absorbed by the surface. Different versions of the report are available including the basic heat gain rate (W), and a per unit area flux (W/m2), and an energy version (J).
+
```

Comment:
Is there any other equipment that emit radiation? OtherEquipment, etc.

Reply:
For EnergyPlus, once solar/visible radiation goes through an *interior* window, it's treated as diffuse on the other side.

Reply:
@mjwitte beat me to it, but that is what I was going to say regarding light that goes through an interior window.  Also, I am pretty certain that nothing else emits visible or SW radiation besides lights.